test/extended/baremetal/common.go

```diff
@@ -91,6 +107,11 @@ func preprovisioningImagesClient(dc dynamic.Interface) dynamic.ResourceInterface
 	return ppiClient.Namespace("openshift-machine-api")
 }
 
+func provisioningClient(dc dynamic.Interface) dynamic.ResourceInterface {
+	pClient := dc.Resource(schema.GroupVersionResource{Group: "metal3.io", Resource: "provisionings", Version: "v1alpha1"})
+	return pClient.Namespace("openshift-machine-api")
```

Comment:
I used without namespace in here and it worked https://github.com/openshift/origin/pull/26832/files#diff-aa7a742599cf4f1743a327027b6221c693e5dfedd562ad3d9c7dd50659ca331aR96.

I think `provisionings`'s scope is cluster wide and no need to use namespace in here.